common/rc

```diff
@@ -542,3 +542,21 @@ _check_conflict_and_set_default()
 
 	eval "${new_name_checked}=true"
 }
+
+_io_uring_enable()
+{
+	# enable io_uring when it is disabled
+	io_uring_disabled=0
```

Comment:
This io_uring_disabled is global. So I suggest to declare it at the top of this script file, and make its name all CAPS.

```
diff --git a/common/rc b/common/rc
index bf9bfe9..01437f4 100644
--- a/common/rc
+++ b/common/rc
@@ -12,6 +12,8 @@ shopt -s extglob
 . common/cgroup
 . common/dm
 
+declare IO_URING_DISABLED
+
 # If a test runs multiple "subtests", then each subtest should typically run
 # for TIMEOUT / number of subtests.
 _divide_timeout() {
@@ -554,17 +556,17 @@ _check_conflict_and_set_default()
 _io_uring_enable()
 {
        # enable io_uring when it is disabled
-       io_uring_disabled=0
+       IO_URING_DISABLED=0
        if [ -f /proc/sys/kernel/io_uring_disabled ]; then
-               io_uring_disabled=$(cat /proc/sys/kernel/io_uring_disabled)
-               if [ "$io_uring_disabled" != 0 ]; then
+               IO_URING_DISABLED=$(cat /proc/sys/kernel/io_uring_disabled)
+               if [ "$IO_URING_DISABLED" != 0 ]; then
                        echo 0 > /proc/sys/kernel/io_uring_disabled
                fi
        fi
 }
 _io_uring_restore()
 {
-       if [ "$io_uring_disabled" != 0 ]; then
-               echo "$io_uring_disabled" > /proc/sys/kernel/io_uring_disabled
+       if [ "$IO_URING_DISABLED" != 0 ]; then
+               echo "$IO_URING_DISABLED" > /proc/sys/kernel/io_uring_disabled
        fi
 }
```